### ISSUE TITLE
Unable to start the plugin

### ISSUE THREAD

**Issue by SinKy-Yan** [lpurdy01/Fusion-360-Game-Controller-Input#1]
When I use Controller_Input_v2, every time I start it, I get a message saying "installing xxxxx" and it launches a new instance of Fusion 360. Do I need to perform some configuration before using it? I noticed the example.env file, but I couldn't find how to configure the directories mentioned in it. My searches on Bing only led me to information on configuring Python directories for Maya.

https://github.com/lpurdy01/Fusion-360-Camera-Control/assets/56128013/651a1110-ae0f-40dc-8cc2-e69fa7ee5549

**Comment by lpurdy01** [comment]
I don't think you have done anything wrong, and  I dont remember having to set anythig up that isnt covered in `lib_installer.install_libs`

I'm setting up a clean computer to see if I can recreate your issue. 

It should open a powershell instance and install the python libraries, haven't seen it start a new instance of fusion before. 


Camera control v2 is dependent on:

```
numpy
pygame
pywinusb
scipy
```

In the mean time see if you can install those manually with info from this guide:

https://dcyoung.github.io/post-f360-py-modules/

**Comment by lpurdy01** [comment]
Please pull the latest commit and see if it works for you. (This is for the os3m mouse, if you don't have one this won't help you. [If you do have one, Hello! excited to meet another!]

I have committed heinous crimes, and included a venv with the Add-In according to Fusion-360's recommended method of including packages. 

https://forums.autodesk.com/t5/fusion-360-api-and-scripts/to-install-python-modules/td-p/5777176

Now instead of attempting to install files it should reference the ones in the venv.

**Comment by lpurdy01** [comment]
I was able to recreate you bug btw.
